packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -60,12 +60,13 @@ export const getOptionStyles = memoizeFunction(
     isPending?: boolean,
     isHidden?: boolean
   ): Partial<IComboBoxOptionStyles> => {
-    const { semanticColors, palette } = theme;
+    const { semanticColors } = theme;
 
-    const ComboBoxOptionBackgroundHovered = semanticColors.menuItemBackgroundHovered;
     const ComboBoxOptionTextColorHovered = semanticColors.bodyText;
-    const ComboBoxOptionTextColorSelected = palette.black;
+    const ComboBoxOptionTextColorSelected = semanticColors.menuItemTextHovered;

```

Comment:
A little bit of a stretch on using this slots on the above 2 tokens. On a hovered token using a regular bodyText and on a selected token using a hovered slot. Not sure about this but not blocking. (cc: @phkuo)

Reply:
yeah it's a little weird, combo box states should match up to menu state, no?
additionally, you can use menuItemText above

Reply:
Yes the ComboBox option should match the menu items specs for all the components... so there could be also some misalignments in the toolkit that a dev thinks that it needs fixing. We need a designer pass over this family of semantic slots and see to which elements they should apply and then see if the toolkit is following them correctly and only then proceed with changes to the code.

Reply:
Thanks you two! Alright so I've changed a few things around. I changed the textColorSelected back to palette.neutralDark since there isnt a semantic slot `menuItemTextChecked/Pressed: neutralDark`. I also changed the textHoveredColor to the correct semanticSlot. @phkuo How do you feel about the new semanticSlot for menuItemChecked/Pressed : p.neutralDark or would you consider this a good place to add it directly to the `input` pool of colors like `inputTextChecked`?

Reply:
If looks like a dropdown menu then it should be part of the dropdown family. Inputs are things like checkboxes, radio buttons, and text input fields.